eng/CodeAnalysis.ruleset

```diff
@@ -174,8 +174,9 @@
     <Rule Id="CA2247" Action="Warning" />       <!-- Argument passed to TaskCompletionSource constructor should be TaskCreationOptions enum instead of TaskContinuationOptions enum. -->
     <Rule Id="CA2248" Action="Warning" />       <!-- Provide correct 'enum' argument to 'Enum.HasFlag' -->
     <Rule Id="CA2249" Action="Warning" />       <!-- Consider using 'string.Contains' instead of 'string.IndexOf' -->
-    <Rule Id="CA2250" Action="Warning" />       <!-- Use 'ThrowIfCancellationRequested' -->
+    <Rule Id="CA2250" Action="None" />          <!-- Use 'ThrowIfCancellationRequested' -->
     <Rule Id="CA2251" Action="Warning" />       <!-- Use 'string.Equals' -->
+    <Rule Id="CA2252" Action="Warning" />       <!-- This API requires opting into preview features -->
```

Comment:
Change this to None? Once I get feedback from Tanner and iterate on this a little more, we can turn it back to Warning?

Reply:
This was supposed to be None.